### PR TITLE
4 style buttons

### LINKED ISSUE #4: Style Buttons
https://www.figma.com/design/JCUTgHjdb6EIA5ZKwEgAOL/WDS---Buttons-and-Links?node-id=801-1001&m=dev

Update decaorateButtons functions to:
- Primary: use STRONG tag, rename it to Brand
- Secondary: use EM tag, rename it to ghost

### PR DESCRIPTION
**Pull request by kesiah**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--edge-delivery-ubs-faweb-demo--netcentric.aem.page/buttons
- After: https://4-style-buttons--edge-delivery-ubs-faweb-demo--netcentric.aem.page/buttons

### CODE REVIEW

**Comment by kesiah** [comment]
Deployment fails because of stylelint, we are using BEM, and stylelint configuration was changed in: https://github.com/Netcentric/edge-delivery-ubs-faweb-demo/pull/11/files#diff-528119d81e5a10738ae49f5e20d333ebc37cea63657eb9069281693e6dae0ad3

File: scripts/aem.js [hunk 1]
```diff
@@ -426,16 +426,23 @@ function decorateButtons(element) {
...
     .find((c) => c.startsWith('icon-'))
     .substring(5);
-  const img = document.createElement('img');
-  img.dataset.iconName = iconName;
-  img.src = `${window.hlx.codeBasePath}${prefix}/icons/${iconName}.svg`;
-  img.alt = alt;
-  img.loading = 'lazy';
-  span.append(img);
+
+  const resp = await fetch(`${window.hlx.codeBasePath}${prefix}/icons/${iconName}.svg`);
```
Comment (dragoss-nc) [inline scripts/aem.js:434]: Is this approach better than adding the path to the svg on the img src?

File: styles/styles.css [hunk 2]
```diff
@@ -244,66 +244,129 @@ a:hover {
...
+a.button.button--secondary,
+button.button--secondary {
+  --button-bg-color: var(--col-white);
+  --button-border-color: var(--col-neutral90);
+  --button-text-color: var(--col-neutral90);
+  --button-icon-color: var(--col-brand40);
+
+  &:hover,
+  &:focus {
+    border-width: 2px;
```
Comment (dragoss-nc) [inline styles/styles.css]: Maybe it would be more consistent to add this as a variable as well.

**Review by dragoss-nc — APPROVED** [review]

File: scripts/aem.js (hunk 1 above)
Comment (kesiah) [reply scripts/aem.js:434]: if we use an `img` we will not be able to change the colors.

**Pull request merged**